executor/common_linux.h

```diff
@@ -5830,3 +5830,64 @@ static long syz_pidfd_open(volatile long pid, volatile long flags)
 }
 
 #endif
+
+#if SYZ_EXECUTOR || __NR_syz_ksmbd_send_req
+#include <netinet/in.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+
+#define KSMBD_BUF_SIZE 16000
+
+static long syz_ksmbd_send_req(volatile long a0, volatile long a1, volatile long a2, volatile long a3)
+{
+	int sockfd;
+	int packet_reqlen;
+	int errno;
+	struct sockaddr_in serv_addr;
+	char packet_req[KSMBD_BUF_SIZE]; // max frame size
+
+	debug("[*]{syz_ksmbd_send_req} entered ksmbd send...\n");
+
+	if (a0 == 0 || a1 == 0) {
+		debug("[!]{syz_ksmbd_send_req} param empty\n");
+		return -7;
```

Comment:
For the fuzzer, it's not really important to distinguish between these specific failure paths. I would just return -1 for all of them.